### ISSUE TITLE
A11y: Custom date range filter options should be grouped

### ISSUE THREAD

**Issue by angelathe** [CDCgov/dibbs-ecr-viewer#1149]
## What needs to be done

Custom date filter form controls in the eCR Library page are not associated with a group label.

## Why it needs to be done

This follows the accessibility audit scans for the eCR Viewer pages.

## Acceptance Criteria

- [ ] Group label should be added to the custom date filter controls in the Library.
- [ ] Manual scans have been run again on these elements, and have come up with no issues.

## Technical Notes

Note: Running the manual guided scans will probably require a Axe pro license. Manual scans for this ticket may specifically require running the `Forms` test for the `Filter by Date` form in the eCR Library.

Axe issue: [Link](https://axe.deque.com/issues/b7096c48-050a-46be-b94f-c4aaab09627c)
[W3C Link: Grouping Controls](https://www.w3.org/WAI/tutorials/forms/grouping/)
[DeQue Axe Issue](https://docs.deque.com/issue-help/1.0.0/en/label-group-not-associated): Group of form controls is not associated with its group label
